src/http/AuthHeader.js

```diff
@@ -85,6 +85,15 @@ class AuthHeader {
         }
         return '';
     }
+
+    static _keyLevel(context) {
```

Comment:
+= comment so we can remember what's going on later
// if we specified that this request should use a keyLevel > LOW, we notice that here
// (and re-set the keyLevel for the next request)

Reply:
added the comment